pkg/loop/internal/relayer/pluginprovider/ext/ccip/offramp_test.go

```diff
@@ -24,8 +24,8 @@ func Test_OnchainConfig(t *testing.T) {
 
 	tests := []struct {
 		name          string
-		onchainConfig ccip.ExecOnchainConfig
-		want          ccippb.ExecOnchainConfig
+		onchainConfig ccip.ExecOnchainConfig   // nolint
+		want          ccippb.ExecOnchainConfig // nolint
```

Comment:
what is the linter unhappy with here?

Reply:
lint doesn't like that the proto is copied because protos have a lock.